django/contrib/auth/password_validation.py

```diff
@@ -0,0 +1,134 @@
+from __future__ import unicode_literals
+
+import os
+import re
+from difflib import SequenceMatcher
+
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured, ValidationError
+from django.template.defaultfilters import unordered_list
+from django.utils.encoding import force_text
+from django.utils.module_loading import import_string
+from django.utils.six import string_types
+from django.utils.translation import ugettext as _
+
+
+def get_password_validators():
```

Comment:
I get your point, but I'm not entirely seeing what you are proposing. Could you make a short example? As this is a more fundamental design discussion, perhaps this would fit best on the django-developers list.

(I'll defer working/responding on other comments until we've settled this issue, as we need to get this right to have a mergeable patch.)